src/common/transactions/transaction-utils.ts

```diff
@@ -38,6 +40,7 @@ export const generateContractCallTx = ({
   return makeContractCall({
     contractName,
     contractAddress,
+    fee: txData.fee !== undefined && isValidFee ? new BN(txData.fee) : undefined,
```

Comment:
stacks.js uses the `in` operator, so this doesn't quite work

![image](https://user-images.githubusercontent.com/1618764/131466551-fd1441d5-6c49-450b-84c8-bc356d906b7c.png)